docs/useful-tips-on-carbondata.md

```diff
@@ -30,16 +30,16 @@
 
   - **Table Column Description**
 
-  | Column Name | Data Type     | Cardinality | Attribution |
-  |-------------|---------------|-------------|-------------|
-  | msisdn      | String        | 30 million  | Dimension   |
-  | BEGIN_TIME  | BigInt        | 10 Thousand | Dimension   |
-  | HOST        | String        | 1 million   | Dimension   |
-  | Dime_1      | String        | 1 Thousand  | Dimension   |
-  | counter_1   | Decimal       | NA          | Measure     |
-  | counter_2   | Numeric(20,0) | NA          | Measure     |
-  | ...         | ...           | NA          | Measure     |
-  | counter_100 | Decimal       | NA          | Measure     |
+| Column Name | Data Type     | Cardinality | Attribution |
+|-------------|---------------|-------------|-------------|
+| msisdn      | String        | 30 million  | Dimension   |
+| BEGIN_TIME  | BigInt        | 10 Thousand | Dimension   |
+| HOST        | String        | 1 million   | Dimension   |
+| Dime_1      | String        | 1 Thousand  | Dimension   |
+| counter_1   | Decimal       | NA          | Measure     |
+| counter_2   | Numeric(20,0) | NA          | Measure     |
+| ...         | ...           | NA          | Measure     |
+| counter_100 | Decimal       | NA          | Measure     |
 
 
   - **Put the frequently-used column filter in the beginning**
```

Comment:
Since we have changed the default behavior of sort_columns, I think this section can be removed. OR we can change it to `Put the fequently-used column filter in the beginning of sort_columns`

Reply:
For the following section, it is the similar with it.

Reply:
updated